model/src/main/scala/clue/data/syntax/package.scala

```diff
@@ -12,4 +16,17 @@ package object syntax {
     def orIgnore: Input[A]   = Input.orIgnore(a)
     def orUnassign: Input[A] = Input.orUnassign(a)
   }
+
+  implicit final class LensToLensOps[S, T, A, B](private val lens: PLens[S, T, A, B])
```

Comment:
Are we using  this method instead of `extension` to remain cross compilable to scala 2?

Reply:
Yes